commands/subcommands/tarkov/item.js

```diff
@@ -1,5 +1,10 @@
 const { request, gql } = require('graphql-request');
 const { EmbedBuilder } = require('discord.js');
+const { Intl } = require('intl')
+
+function formatNumber(num) {
```

Comment:
May not come up, but would be ideal to make a 'helpers.js' file and export this so we can use it in another file if need be. DRY that shit dawg.

Reply:
Also may be worth making it work for USD when you make a helper function. 

So the arguments would be (num, format)

And depending on format, it would return RUB, USD, Euro, Etc.